docs/photoniq/faas/03-faas-commands/03-faas-cli-commands.md

```diff
@@ -212,20 +214,20 @@ Execute a function on a remote server, optionally specifying a version or alias.
 **Syntax:**
 
 ```bash
-faas remote execute <fn-name> [<version>|<alias>] --data <data-file>
+faas remote execute <fn-name> [<version>|<alias>] --file data.json
 ```
 
 **Examples:**
 
 ```bash
   # Execute the latest version of a remote function with data
-  faas remote execute "my-function" --data data.json
+  faas remote execute "my-function" --file data.json
 
   # Execute a specific version of a remote function with data
-  faas remote execute "my-function" "1.0.0" --data data.json
+  faas remote execute "my-function" "1.0.0" --file data.json
 
   # Execute a remote function by alias with data
-  faas remote execute "my-function" "dev" --data data.json
+  faas remote execute "my-function" "dev" --file data.json
 ```
```

Comment:
I have the same question here. Specifying a data.json file almost says that's the only accepted file format, but I remember Felipe mentioning that it accepts other data files.